apitools/base/py/base_api.py

```diff
@@ -376,6 +378,15 @@ def num_retries(self, value):
         self.__num_retries = value
 
     @property
+    def check_response_func(self):
```

Comment:
Setter seems useful because you can modify the client temporarily for a single call or a class of calls.